.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+# Workflow for updating the documentation website
+---
+name: Update Documentation
+
+# Documentation updates can happen manually or within a release; allow only one at a time.
+concurrency: release
+
+on:
+  # Allow running this workflow manually from the Actions tab
+  workflow_dispatch:
+  # Allow this to be a reusable workflow, but only meant to be called from within the same repository
+  # Reference: https://docs.github.com/en/actions/using-workflows/reusing-workflows
+  workflow_call:
+    secrets:
+      GH_RELEASE_PAT:
+        required: true
+
+jobs:
+  Update-Documentation:
+    name: Update the documentation
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    environment:
+      name: github-pages
+      url: https://cli.phylum.io
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Configure git user
+        run: |
+          git config user.name 'phylum-bot'
+          git config user.email '69485888+phylum-bot@users.noreply.github.com'
+
+      - name: Set up Node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: 18
```

Comment:
Another concrete node version here.

Reply:
This will be updated with one of the proposed solutions for staying fast and DRY.